packages/api/src/module/write/shared/application/events-subscription/events-subscription.spec.ts

```diff
@@ -95,6 +97,7 @@ describe('Events subscription', () => {
       })
       .mockImplementationOnce(() => {}) // 7. event #4 - success (retry: 3)
       .mockImplementationOnce(() => {
+        subscription.stop(); // force stop processing - simulate no retries
```

Comment:
Why do we need it? Max retries for queue in config is 3. 
What is max for single event?

Reply:
In reality, we have two retry strategies:

1. retry on queue in `listen()` phase to synchronize order of events ([short fixed delay](https://github.com/CodersCrew/coderscamp/pull/373/files#diff-549bcd44f6766e2dfd8c0ab2648924875a3972b0a52282e97796215f4120324eR231))
2. retry when sth goes wrong during events processing (e.g. DB connection lose, handleEvent throws an exception, etc.) ([configurable delay](https://github.com/CodersCrew/coderscamp/pull/373/files#diff-549bcd44f6766e2dfd8c0ab2648924875a3972b0a52282e97796215f4120324eR274-R295))

I removed number of retries from the second strategy. Now, EventsSubscriber works until specific condition. ( in our case: until stop/shutdown request ). I didn't wanna change this test so I decided to simulate stop processing after 3 retries.